projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/DataJobDefaultConfigurations.java

```diff
@@ -31,4 +33,32 @@ public KubernetesService.Resources dataJobRequests() {
   public KubernetesService.Resources dataJobLimits() {
     return new KubernetesService.Resources(cpuLimits, memoryLimits);
   }
+
+  public float getCpuRequests() throws ParseException {
+    return NumberFormat.getInstance().parse(cpuRequests).floatValue();
+  }
+
+  public float getCpuLimits() throws ParseException {
+    return NumberFormat.getInstance().parse(cpuLimits).floatValue();
+  }
+
+  public int getMemoryRequests() throws ParseException {
+    var memoryRequest = NumberFormat.getInstance().parse(memoryRequests).intValue();
+    return getMemoryInMi(memoryRequests, memoryRequest);
+  }
+
+  public int getMemoryLimits() throws ParseException {
+    var memoryLimit = NumberFormat.getInstance().parse(memoryLimits).intValue();
+    return getMemoryInMi(memoryLimits, memoryLimit);
+  }
+
+  private int getMemoryInMi(String memory, int amount) {
+    if (memory.endsWith("Gi")) {
+      return amount * 1024;
+    }
+    if (memory.endsWith("G")) {
+      return amount * 1000;
+    }
+    return amount;
+  }
```

Comment:
Ideally, our code should account for a wide range of expected scenarios. In this case this includes accommodating different memory units such as MB, KB, and even TB, in addition to GB. Users might need these variations, and our code should be flexible enough to handle them.

Also It seems like this type of function is a prime candidate for a Utility class, given its very reusable nature.